crates/rnote-ui/src/dialogs/mod.rs

```diff
@@ -552,6 +553,38 @@ pub(crate) async fn dialog_edit_selected_workspace(appwindow: &RnAppWindow) {
     dialog.present(appwindow);
 }
 
+pub(crate) async fn dialog_trash_file(appwindow: &'static RnAppWindow, filerow: &RnFileRow) -> () {
```

Comment:
You probably don't need the `&'static` here

Reply:
Sucess : 
<img width="828" alt="Screenshot 2024-08-27 at 22 30 07" src="https://github.com/user-attachments/assets/67cfc8f2-36df-4ecc-8413-68d0f87b3403">

Reply:
Wow, thanks for the hint. I added that when trying to fix another issue and totally forgot about it. Let me fix this up and then mark this ready for review. Tested it and it seems to work fine :)